index.html

```diff
@@ -362,7 +367,62 @@ <h2>Project related news</h2>
               <p class="news-date">13 October 2017</p>
             </div>
           </div>
-
+        </div>
+        <div class="row">
+          <div class="col-xs-12 col-sm-4 col-md-4 col-lg-4">
+            <div class="news-wrapper">
+              <img src="img/news.png" alt="news icon" class="news-icon" />

```

Comment:
Which is a news icon and which is a website?  News items for Herald, Scoop, Stuff, ODT. Website item for the law commission website.

Reply:
Cool ok

Reply:
Hey - All the links are news items should we use the news icon if they are all the same type? The law commission link up to the part where it talks about the Law Commission submission stuff (no point duplicating it here and also it throws out the symmetry!)